windows/ReactNativeVideoCPP/ReactVideoView.cpp

```diff
@@ -13,15 +13,24 @@ using namespace Windows::UI::Xaml::Controls;
 using namespace Windows::UI::Core;
 using namespace Windows::Media::Core;
 using namespace Windows::Media::Playback;
+using namespace Microsoft::UI::Xaml::Controls;
 
 namespace winrt::ReactNativeVideoCPP::implementation {
 
 ReactVideoView::ReactVideoView(winrt::Microsoft::ReactNative::IReactContext const &reactContext)
     : m_reactContext(reactContext) {
+  // Create MediaPlayerElement using Microsoft.UI.Xaml.Controls
+  m_mediaPlayerElement = MediaPlayerElement();
+  
   // always create and set the player here instead of depending on auto-create logic
   // in the MediaPlayerElement (only when auto play is on or URI is set)
   m_player = winrt::Windows::Media::Playback::MediaPlayer();

```

Comment:
```suggestion
    : m_reactContext(reactContext)
    // Create MediaPlayerElement using Microsoft.UI.Xaml.Controls
    , m_mediaPlayerElement(MediaPlayerElement())
    // always create and set the player here instead of depending on auto-create logic
    // in the MediaPlayerElement (only when auto play is on or URI is set)
    , m_player(winrt::Windows::Media::Playback::MediaPlayer()) {
```

Use initializer lists, not assignment.  Refer [Should my constructors use “initialization lists” or “assignment”?](https://isocpp.org/wiki/faq/ctors#init-lists)